nautobot/utilities/testing/views.py

```diff
@@ -1021,7 +1022,10 @@ class BulkDeleteObjectsViewTestCase(ModelViewTestCase):
         """
 
         @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
-        def test_bulk_delete_objects_without_permission(self):
+        @patch("nautobot.core.views.generic.BulkDeleteView.perform_pre_delete")
+        def test_bulk_delete_objects_without_permission(self, mock_perform_pre_delete):
+            if self.model == CustomField:
+                mock_perform_pre_delete.return_value = None
```

Comment:
Thanks for adding these, but instead of doing this here at the generic test code, please instead overload these methods on `nautobot.extras.tests.test_views.CustomFieldTestCase`. So for example, this one would look like this on that class:

```python
        @override_settings(EXEMPT_VIEW_PERMISSIONS=[])
        @patch("nautobot.core.views.generic.BulkDeleteView.perform_pre_delete")
        def test_bulk_delete_objects_without_permission(self, mock_perform_pre_delete):
            mock_perform_pre_delete.return_value = None
            return super().test_bulk_delete_objects_without_permission()
```